plugins/jump/jump.plugin.zsh

```diff
@@ -9,7 +9,7 @@
 export MARKPATH=$HOME/.marks
 
 jump() {
-	cd -P "$MARKPATH/$1" 2>/dev/null || echo "No such mark: $1"
+	cd -P "$MARKPATH/$1" 2>/dev/null || (echo "No such mark: $1"; return 1)
```

Comment:
A little style fix: use braces `{}` instead of spaces `()` which runs it in a subshell, with the added cost of creating a new process. The rest if perfect :+1: